contracts/CourtRoom.sol

```diff
@@ -37,7 +45,6 @@ contract CaseContract is Owned {
 		return clm.claimId;
 	}
```

Comment:
It only for the case where the claim is valid and the client is compensated .After this case there is no guaranty  that there is enough deposit left for all registered players.So a re registration need to be done to assure that.